docs/admin/install.rst

```diff
@@ -346,8 +353,8 @@ It is usually a good idea to run Weblate in a separate database, and separate us
 .. hint::
 
    If you don't want to make the Weblate user a superuser in PostgreSQL, you can
-   omit that. In that case you will have to perform some of the migration steps
-   manually as a PostgreSQL superuser in schema Weblate will use:
+   omit that and perform some of the migration steps manually.
+   This is an example of a PostgreSQL superuser in the scheme Weblate uses:
```

Comment:
Not sure here

Reply:
But that is no way to live. I am however 100% sure I will forget to mention it. This is the one I thought was more ready to go.